app/assets/javascripts/about.js.coffee

```diff
@@ -0,0 +1,9 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
+
+object = {
+  spacing: true,
+  spacing: true,
+  spacing: true,
```

Comment:
__[development][coffeelint]__
* [error] Duplicate key defined in object or class